### ISSUE TITLE
Issue while trying to execute the Speechcommands V2 Recipe

### ISSUE THREAD

**Issue by alexandramarkal** [YuanGongND/ast#140]
I have tried a lot of times to execute the script and every time it gets stucked in this point
+ python -W ignore ../../src/run.py --model ast --dataset speechcommands --data-train ./data/datafiles/speechcommand_train_data.json --data-val ./data/datafiles/speechcommand_valid_data.json --data-eval ./data/datafiles/speechcommand_eval_data.json --exp-dir ./exp/test-speechcommands-f10-t10-pTrue-b128-lr2.5e-4-decoupe --label-csv ./data/speechcommands_class_labels_indices.csv --n_class 35 --lr 2.5e-4 --n-epochs 30 --batch-size 128 --save_model True --freqm 48 --timem 48 --mixup 0.6 --bal none --dataset_mean -6.845978 --dataset_std 5.5654526 --audio_length 128 --noise True --metrics acc --loss BCE --warmup False --lrscheduler_start 5 --lrscheduler_step 1 --lrscheduler_decay 0.85 --tstride 10 --fstride 10 --imagenet_pretrain True --audioset_pretrain False

Downloading: "https://dl.fbaipublicfiles.com/deit/deit_base_distilled_patch16_384-d0272ac0.pth" to ../../pretrained_models/hub/checkpoints/deit_base_distilled_patch16_384-d0272ac0.pth

I'm searching everywhere for this url/model : https://dl.fbaipublicfiles.com/deit/deit_base_distilled_patch16_384-d0272ac0.pth and I couldn't find anywhere in the code.. 
Thank you in advance.

**Comment by alexandramarkal** [comment]
Problem solved! I just downloaded manually (using "wget https://dl.fbaipublicfiles.com/deit/deit_base_distilled_patch16_384-d0272ac0.pth") the pretrained weights under this path "../../pretrained_models/hub/checkpoints" . Once the pre-trained weights are in place, the AST model initializes its parameters with these weights.
